build/release.sh

```diff
@@ -78,7 +78,7 @@ reset_to_master() {
 
 build_lint_and_test() {
     # The build command includes linting
-    cmd yarn build && cmd yarn test || return 1
+    cmd yarn build && cmd yarn test:ci || return 1
```

Comment:
If we ever need to run this locally, will this still work?

Reply:
Yep, it just may be a little slower due to limited maxWorkers.